OpenOversight/app/utils.py

```diff
@@ -290,9 +294,13 @@ def upload_obj_to_s3(file_obj, dest_filename):
 
 
 def filter_by_form(form_data, officer_query, department_id=None):
-    if form_data.get("name"):
+    if form_data.get("last_name"):
+        officer_query = officer_query.filter(
+            Officer.last_name.ilike("%%{}%%".format(form_data["last_name"]))
+        )
+    if form_data.get("first_name"):
         officer_query = officer_query.filter(
-            Officer.last_name.ilike("%%{}%%".format(form_data["name"]))
+            Officer.first_name.ilike("%%{}%%".format(form_data["first_name"]))
```

Comment:
It'd be nice if `"%%{}%%"` was in a constant variable because:
1. It's used more than once.
2. To give the string some meaning.

Reply:
Hmm, something doesn't look right about this. Not sure why we need two percentage signs here. I think an f-string would be better here (now that we finally have python 3.8). So something like `Officer.first_name.ilike(f"%{form_data["first_name"]}%")` I think I would prefer